scipy/stats/tests/test_marray.py

```diff
@@ -0,0 +1,126 @@
+import pytest
+import numpy as np
+from scipy import stats
+
+from scipy._lib._array_api import xp_assert_close, xp_assert_equal
+from scipy.stats._stats_py import _xp_mean, _xp_var
+skip_backend = pytest.mark.skip_xp_backends
+
+
+def get_arrays(n_arrays, *, dtype='float64', xp=np, shape=(7, 8), seed=84912165484321):
+    marray = pytest.importorskip('marray')
+    xpm = marray._get_namespace(xp)
```

Comment:
nit for consistency (here and below)
```suggestion
    mxp = marray._get_namespace(xp)
```

Reply:
Ok. I did find myself going back and forth. I think I prefer `xpm` but found myself doing `mxp` sometimes. You have a preference?

Reply:
I have a preference for `mxp`. I think we've discussed this before, but my mental model of library-wrapping like this goes left-to-right as it goes outside-in, so e.g. `marray(dask(numpy))` rather than `((numpy)dask)marray)`.

Reply:
I don't think one could disagree with the mental model of namespace composition, of course. But that is not the only factor one might want to represent with the reference name. I think it's more important to emphasize `xp`, which means to me that it's standing in for *any* *Py*thon array API compatible backend, than the fact that it's a masked version of the namespace. `m` might as well be a subscript of `xp` to denoting the fact that it's a masked version. (That doesn't mean I'd advocate for `xp_m`, though.)

I think it's similar to:
```python3
import numpy as np
import cupy as cp
x = [1, 2, 3, 4, 5]
x_np = np.asarray(x)
x_cp = cp.asarray(x)
```
One could argue that a NumPy/CuPy array was created from `x` and therefore it should be `np_x`/`cp_x`, but I think the fact that the variable still represents `x` is the important thing and therefore belongs first. Would you advocate for `np_x`/`cp_x`?

Anyway, it's not super important, so I'll change it to `mxp` here. I'm just explaining that my preference isn't arbitrary, either, and I've thought about changing it in the `marray` documentation.

Reply:
Well, I'm not sure. I think somebody might have used `npm` before, though 😅